numpy/lib/tests/test_nanfunctions.py

```diff
@@ -83,6 +83,17 @@ def test_result_values(self):
             res = nf(_ndat, axis=1)
             assert_almost_equal(res, tgt)
 
+    def test_initializer(self):
+        mat1 = np.array([np.nan] * 9).reshape(3, 3)
+        mat2 = mat1.copy()
+        mat2[1] = 1
+
+        assert_equal(np.nanmin(mat1, initializer=5), 5.0)
+        assert_equal(np.nanmax(mat1, initializer=6), 6.0)
+
+        assert_equal(np.nanmin(mat2, initializer=5), 1.0)
+        assert_equal(np.nanmax(mat2, initializer=6), 6.0)
```

Comment:
Let's not use `mat` to avoid confusion with `np.matrix`. `arr` or `a` is pretty typical in tests

Reply:
Done.